src/tests/suite/daos_container.c

```diff
@@ -2203,6 +2204,52 @@ co_rf_simple(void **state)
 	test_teardown((void **)&arg);
 }
 
+static void
+delet_container_during_aggregation(void **state)
+{
+	test_arg_t	*arg = *state;
+	daos_obj_id_t	 oid;
+	daos_pool_info_t pinfo;
+	int		i;
+	int		rc;
+
+	/* Prepare records */
+	oid = dts_oid_gen(OC_RP_2G1, 0, arg->myrank);
```

Comment:
That implies the test can only run on nodes >= 2, maybe we can use OC_SX that the test can run on single node as well.

Reply:
oh sure.